sympy/stats/joint_rv.py

```diff
@@ -233,12 +233,24 @@ def _sample_pymc3(cls, dist, size, seed):
         if dist.__class__.__name__ not in dist_list:
             return None
 
+        #size accepts only int in case of pymc3
+        #see https://github.com/sympy/sympy/pull/21590#discussion_r649783860
+
+        draws = list(size)[1:]
+        if draws == []:
+            draws=1
+        elif len(draws) == 1:
+            draws = draws[0]
+        else:
+            raise TypeError("Invalid value for `size` in pymc3. Must be int")
+        import logging
+        logging.getLogger("pymc3").setLevel(logging.ERROR)
         with pymc3.Model():
             pymc3_rv_map[dist.__class__.__name__](dist)
-            samples = pymc3.sample(size, chains=1, progressbar=False, random_seed=seed)[:]['X']
-            if samples.shape[0:len(size)] != size:
-                samples = samples.reshape((size[0],) + samples.shape)
-            return samples
+            samples = pymc3.sample(draws=draws, chains=1, progressbar=False, random_seed=seed, return_inferencedata=False, compute_convergence_checks=False)[:]['X']
+        if samples.shape[0:len(size)] != size:
+            samples = samples.reshape((size[0],) + samples.shape)
+        return samples
```

Comment:
If we're going to call `reshape` here then we can presumably allow size to be a tuple. Can it just be something like:
```
draws = prod(size)
samples = pymc3.sample(draws=prod(size), ...)
samples = samples.reshape(size)
```

Reply:
Thats a great idea! It could even make something like this possible for pymc3,
```python
>>> sample(N, library='pymc3', size=(2,2,3))
array([[[11.72402844, 11.72402844, 11.72402844],
        [ 8.20910153,  8.72181852,  8.72181852]],

       [[ 8.72181852,  9.78352304,  9.78352304],
        [ 9.78352304,  9.78352304,  9.78352304]]])
```